src/main/java/net/minestom/server/command/builder/arguments/minecraft/registry/ArgumentPotionEffect.java

```diff
@@ -14,7 +14,8 @@ public ArgumentPotionEffect(String id) {
 
     @Override
     public String parser() {
-        return "minecraft:mob_effect";
+        // TODO: what replace `minecraft:mob_effect`?
```

Comment:
?

Reply:
Mob effect has been removed in 1.19.3, i think it should be replaced with string or removed from minestom

Reply:
For now, it will be removed